server/src/vcap-constants.es6

```diff
@@ -93,6 +98,13 @@ vcapConstants.PAY_GOV_APP_ID = payGov.tcs_app_id;
 vcapConstants.PAY_GOV_CERT = payGov.certificate;
 vcapConstants.PAY_GOV_PRIVATE_KEY = payGov.private_key;
 
+/** Database configuration */
+const database = getUserProvided('database') || {};
```

Comment:
Yep, exactly. If you specify the database url by environment variable, the ssl configuration is determined by NODE_ENV (ssl: true for production). So on cloud.gov, (staging and production), database is set by URL and it will connect over ssl.